docs/getting_started/requirements.md

```diff
@@ -20,7 +20,7 @@ Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached,
 ## Operating system
 
 - Debian 10.x "buster" or Debian 11.x "bullseye"
-- Ubuntu 20.04 "Focal Fossa"
+- Ubuntu 20.04 "Focal Fossa" or Ubuntu 22.04 LTS "Jammy Jellyfish"
```

Comment:
20.04 is also a LTS. 
Do we want to make it consistent? 🙂 
Add it in both? Skip it in both?

Reply:
It seems in case of Debian we do skip it - ref. https://wiki.debian.org/LTS.